src/objective_coefficients.jl

```diff
@@ -26,9 +26,9 @@ struct PrimalObjective{T}
 
     function PrimalObjective{T}(obj::SAF{T}) where T
         canonical_obj = MOIU.canonical(obj)
-        if isempty(canonical_obj.terms)
-            error("Dualization does not support models with no variables in the objective function.")
-        end
+        # if isempty(canonical_obj.terms)
+        #     error("Dualization does not support models with no variables in the objective function.")
+        # end
```

Comment:
Could we pass another argument here so we identify if the primal objective has no terms or if the terms were just ignored? This would avoid getting weird results in feasibility problems defined as `min 0 * x` .

Reply:
Thats a possibility.
What could go wrong if we have a problem that is min 0 ?

Reply:
A normal user would try to dualize a feasibility problem unintentionally and we would not give him a warn/error that this is not a possibility.

Reply:
I was thinking...
`min 0 * x` is a minimization problem, and would have a well posed dual. It would be perfectly compatible with the JuMP definitions.
I agree we should protect against the `FEASIBILITY_SENSE`, but that is done already right?

Reply:
I can imagine someone solving a sequence of parametric LPs where some of them have zero as coefficients for the objective function.

Reply:
Ok, in my mind that would be a hidden feasibility problem. We do have a treatment for FEASIBILITY_SENSE.

Reply:
My reasoning is: JuMP is `FEASIBILITY_SENSE` by default. So if it is `MIN_SENSE`, the  use explicitly added it, and everything would be well posed.
The this with `FEASIBILITY_SENSE` is that there is no dual right? but a MIN problem with zero objective e has a well defined dual.

Reply:
min 0 seems like the right interpretation.

Reply:
Ok! :+1: